CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+3.0.0
+---
+* Move transaction id management and module persistence to PactDb backend
+* PactValue type for reliable Term output with user-friendly JSON
+* Better compile errors
+* Fixes Pact server persistence bug
+* Adds `with-default-read`
+* Adds SPV support with configurable backend
+* Removes support for non-string keys in objects
+* Uses cryptonite for Windows build
```

Comment:
Confirm we have a working windows build with @vaibhavsagar and change to "Working windows build" -- do we need more work to actually offer/support this?